app/lib/features/tasks/sheets/create_update_task_list.dart

```diff
@@ -75,43 +75,40 @@ class _CreateUpdateTaskListConsumerState
   Widget _buildBody(BuildContext context) {
     return Padding(
       padding: const EdgeInsets.symmetric(horizontal: 20),
-      child: SingleChildScrollView(
-        child: Form(
-          key: _formKey,
-          child: Column(
-            mainAxisSize: MainAxisSize.min,
-            crossAxisAlignment: CrossAxisAlignment.stretch,
-            children: [
-              const SizedBox(height: 20),
-              const Divider(
-                indent: 150,
-                endIndent: 150,
-                thickness: 2,
-              ),
-              Text(
-                L10n.of(context).createNewTaskList,
-                textAlign: TextAlign.center,
-                style: Theme.of(context).textTheme.titleMedium,
-              ),
-              const SizedBox(height: 40),
-              Center(
-                child: ActerIconWidget(
-                  onIconSelection: (taskListIconColor, taskListIcon) {
-                    this.taskListIconColor = taskListIconColor;
-                    this.taskListIcon = taskListIcon;
-                  },
+      child: Container(
+        padding: MediaQuery.of(context).viewInsets,
```

Comment:
Nothing much change here, Just added to single line to resolve keyboard overlapping issue on Add Task List Bottom Sheet.